cg/meta/report/api.py

```diff
@@ -162,6 +163,19 @@ def _incorporate_coverage_data(self, samples: list, panels: list):
             else:
                 self.LOG.warning(f"No coverage could be calculated for: {lims_id}")
 
+    def _check_sample_dates(self, sample, delivery_data_sample):
+        prep_category = sample.application_version.application.prep_category
+        for status in ["received", "prep_date", "sequencing_date", "delivery_date"]:
+            if prep_category == "rml" and status == "prep_date":
```

Comment:
Do we create delivery report for rml samples? otherwise I suggest we remove this check

Reply:
Do you know @emmser ?

Reply:
Not now, but we have to add it in the future